Adding Assumable ID example for AWS EC2

SharpRake opened the pull request:
[ ] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->
This PR adds a new assumable identity example, this one for an EC2 instance.

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it. -->
Resolves https://github.com/chainguard-dev/internal/issues/4608

### Why are we making this change?
<!-- What larger problem does this PR address? -->

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please add them. -->
@manojkrgupta wrote the original draft of this - I went a little nuts with rewriting things so please make sure nothing important was lost in translation.

@max-allan-cgr was flagged in the original PR, his review would be appreciated, though not required.

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps. -->
I've tested the process outlined thoroughly, but I'm worried the approach I landed on might be a little confusing. Please run through the example with an EC2 instance and let me know your thoughts.

[preview link](https://deploy-preview-2117--ornate-narwhal-088216.netlify.app/chainguard/administration/assumable-ids/identity-examples/aws-ec2-identity/)

SharpRake committed:
Adding Assumable ID example for AWS EC2

Signed-off-by: Mark Drake <mark@chainguard.dev>

SharpRake committed:
minor tweaks

Signed-off-by: Mark Drake <mark@chainguard.dev>

dependabot[bot] committed:
Bump the actions group with 2 updates (#2119)

Bumps the actions group with 2 updates:
[actions/setup-node](https://github.com/actions/setup-node) and
[docker/login-action](https://github.com/docker/login-action).

Updates `actions/setup-node` from 4.2.0 to 4.3.0
<details>
<summary>Release notes</summary>
<p><em>Sourced from <a
href="https://github.com/actions/setup-node/releases">actions/setup-node's
releases</a>.</em></p>
<blockquote>
<h2>v4.3.0</h2>
<h2>What's Changed</h2>
<h3>Dependency updates</h3>
<ul>
<li>Upgrade <code>@​actions/glob</code> from 0.4.0 to 0.5.0 by <a
href="https://github.com/dependabot"><code>@​dependabot</code></a> in <a
href="https://redirect.github.com/actions/setup-node/pull/1200">actions/setup-node#1200</a></li>
<li>Upgrade <code>@​action/cache</code> from 4.0.0 to 4.0.2 by <a
href="https://github.com/gowridurgad"><code>@​gowridurgad</code></a> in
<a
href="https://redirect.github.com/actions/setup-node/pull/1251">actions/setup-node#1251</a></li>
<li>Upgrade <code>@​vercel/ncc</code> from 0.38.1 to 0.38.3 by <a
href="https://github.com/dependabot"><code>@​dependabot</code></a> in <a
href="https://redirect.github.com/actions/setup-node/pull/1203">actions/setup-node#1203</a></li>
<li>Upgrade <code>@​actions/tool-cache</code> from 2.0.1 to 2.0.2 by <a
href="https://github.com/dependabot"><code>@​dependabot</code></a> in <a
href="https://redirect.github.com/actions/setup-node/pull/1220">actions/setup-node#1220</a></li>
</ul>
<h2>New Contributors</h2>
<ul>
<li><a
href="https://github.com/gowridurgad"><code>@​gowridurgad</code></a>
made their first contribution in <a
href="https://redirect.github.com/actions/setup-node/pull/1251">actions/setup-node#1251</a></li>
</ul>
<p><strong>Full Changelog</strong>: <a
href="https://github.com/actions/setup-node/compare/v4...v4.3.0">https://github.com/actions/setup-node/compare/v4...v4.3.0</a></p>
</blockquote>
</details>
<details>
<summary>Commits</summary>
<ul>
<li><a
href="https://github.com/actions/setup-node/commit/cdca7365b2dadb8aad0a33bc7601856ffabcc48e"><code>cdca736</code></a>
Bump <code>@​actions/tool-cache</code> from 2.0.1 to 2.0.2 (<a
href="https://redirect.github.com/actions/setup-node/issues/1220">#1220</a>)</li>
<li><a
href="https://github.com/actions/setup-node/commit/22c0e7494f4cf8e8133097a3fefafe255caf66eb"><code>22c0e74</code></a>
Bump <code>@​vercel/ncc</code> from 0.38.1 to 0.38.3 (<a
href="https://redirect.github.com/actions/setup-node/issues/1203">#1203</a>)</li>
<li><a
href="https://github.com/actions/setup-node/commit/a7c2d9473e135474f9c8e6d48969d1d90f950ea1"><code>a7c2d94</code></a>
actions/cache upgrade (<a
href="https://redirect.github.com/actions/setup-node/issues/1251">#1251</a>)</li>
<li><a
href="https://github.com/actions/setup-node/commit/802632921f8532d2409ae6eac3313b6f81f11122"><code>8026329</code></a>
Bump <code>@​actions/glob</code> from 0.4.0 to 0.5.0 (<a
href="https://redirect.github.com/actions/setup-node/issues/1200">#1200</a>)</li>
<li>See full diff in <a
href="https://github.com/actions/setup-node/compare/1d0ff469b7ec7b3cb9d8673fde0c81c44821de2a...cdca7365b2dadb8aad0a33bc7601856ffabcc48e">compare
view</a></li>
</ul>
</details>
<br />

Updates `docker/login-action` from 3.3.0 to 3.4.0
<details>
<summary>Release notes</summary>
<p><em>Sourced from <a
href="https://github.com/docker/login-action/releases">docker/login-action's
releases</a>.</em></p>
<blockquote>
<h2>v3.4.0</h2>
<ul>
<li>Bump <code>@​actions/core</code> from 1.10.1 to 1.11.1 in <a
href="https://redirect.github.com/docker/login-action/pull/791">docker/login-action#791</a></li>
<li>Bump <code>@​aws-sdk/client-ecr</code> to 3.766.0 in <a
href="https://redirect.github.com/docker/login-action/pull/789">docker/login-action#789</a>
<a
href="https://redirect.github.com/docker/login-action/pull/856">docker/login-action#856</a></li>
<li>Bump <code>@​aws-sdk/client-ecr-public</code> to 3.758.0 in <a
href="https://redirect.github.com/docker/login-action/pull/789">docker/login-action#789</a>
<a
href="https://redirect.github.com/docker/login-action/pull/856">docker/login-action#856</a></li>
<li>Bump <code>@​docker/actions-toolkit</code> from 0.35.0 to 0.57.0 in
<a
href="https://redirect.github.com/docker/login-action/pull/801">docker/login-action#801</a>
<a
href="https://redirect.github.com/docker/login-action/pull/806">docker/login-action#806</a>
<a
href="https://redirect.github.com/docker/login-action/pull/858">docker/login-action#858</a></li>
<li>Bump cross-spawn from 7.0.3 to 7.0.6 in <a
href="https://redirect.github.com/docker/login-action/pull/814">docker/login-action#814</a></li>
<li>Bump https-proxy-agent from 7.0.5 to 7.0.6 in <a
href="https://redirect.github.com/docker/login-action/pull/823">docker/login-action#823</a></li>
<li>Bump path-to-regexp from 6.2.2 to 6.3.0 in <a
href="https://redirect.github.com/docker/login-action/pull/777">docker/login-action#777</a></li>
</ul>
<p><strong>Full Changelog</strong>: <a
href="https://github.com/docker/login-action/compare/v3.3.0...v3.4.0">https://github.com/docker/login-action/compare/v3.3.0...v3.4.0</a></p>
</blockquote>
</details>
<details>
<summary>Commits</summary>
<ul>
<li><a
href="https://github.com/docker/login-action/commit/74a5d142397b4f367a81961eba4e8cd7edddf772"><code>74a5d14</code></a>
Merge pull request <a
href="https://redirect.github.com/docker/login-action/issues/856">#856</a>
from docker/dependabot/npm_and_yarn/aws-sdk-dependenc...</li>
<li><a
href="https://github.com/docker/login-action/commit/2f4f00e4c6fe8a50cdd1fd618421be2e92b2f201"><code>2f4f00e</code></a>
chore: update generated content</li>
<li><a
href="https://github.com/docker/login-action/commit/67c184546cf989af16f02a1b5359e4bde3cdc524"><code>67c1845</code></a>
build(deps): bump the aws-sdk-dependencies group across 1 directory with
2 up...</li>
<li><a
href="https://github.com/docker/login-action/commit/3d4cc89e85e0cac73870ab81d3b72c0b700870d1"><code>3d4cc89</code></a>
Merge pull request <a
href="https://redirect.github.com/docker/login-action/issues/844">#844</a>
from graysonpike/master</li>
<li><a
href="https://github.com/docker/login-action/commit/6cc823a6c4738f797f031790fa7f982b7a8dcfdc"><code>6cc823a</code></a>
Merge pull request <a
href="https://redirect.github.com/docker/login-action/issues/823">#823</a>
from docker/dependabot/npm_and_yarn/proxy-agent-depen...</li>
<li><a
href="https://github.com/docker/login-action/commit/d94e792124647378e94c07359922f0f821a9fab2"><code>d94e792</code></a>
chore: update generated content</li>
<li><a
href="https://github.com/docker/login-action/commit/033db0da3047b4d01e249d66f56ae16a2ed6af87"><code>033db0d</code></a>
Merge pull request <a
href="https://redirect.github.com/docker/login-action/issues/812">#812</a>
from docker/dependabot/github_actions/codecov/codecov...</li>
<li><a
href="https://github.com/docker/login-action/commit/09c2ae9716c0bacef3c03e120a61c28adfb8595b"><code>09c2ae9</code></a>
build(deps): bump https-proxy-agent</li>
<li><a
href="https://github.com/docker/login-action/commit/ba56f006fc7190f752d4e6e1312f85697984a229"><code>ba56f00</code></a>
ci: update deprecated input for codecov-action</li>
<li><a
href="https://github.com/docker/login-action/commit/75bf9a79af089e9aa009972a6ecb22190a520679"><code>75bf9a7</code></a>
Merge pull request <a
href="https://redirect.github.com/docker/login-action/issues/858">#858</a>
from docker/dependabot/npm_and_yarn/docker/actions-to...</li>
<li>Additional commits viewable in <a
href="https://github.com/docker/login-action/compare/9780b0c442fbb1117ed29e0efdff1e18412f7567...74a5d142397b4f367a81961eba4e8cd7edddf772">compare
view</a></li>
</ul>
</details>
<br />


Dependabot will resolve any conflicts with this PR as long as you don't
alter it yourself. You can also trigger a rebase manually by commenting
`@dependabot rebase`.

[//]: # (dependabot-automerge-start)
[//]: # (dependabot-automerge-end)

---

<details>
<summary>Dependabot commands and options</summary>
<br />

You can trigger Dependabot actions by commenting on this PR:
- `@dependabot rebase` will rebase this PR
- `@dependabot recreate` will recreate this PR, overwriting any edits
that have been made to it
- `@dependabot merge` will merge this PR after your CI passes on it
- `@dependabot squash and merge` will squash and merge this PR after
your CI passes on it
- `@dependabot cancel merge` will cancel a previously requested merge
and block automerging
- `@dependabot reopen` will reopen this PR if it is closed
- `@dependabot close` will close this PR and stop Dependabot recreating
it. You can achieve the same result by closing it manually
- `@dependabot show <dependency name> ignore conditions` will show all
of the ignore conditions of the specified dependency
- `@dependabot ignore <dependency name> major version` will close this
group update PR and stop Dependabot creating any more for the specific
dependency's major version (unless you unignore this specific
dependency's major version or upgrade to it yourself)
- `@dependabot ignore <dependency name> minor version` will close this
group update PR and stop Dependabot creating any more for the specific
dependency's minor version (unless you unignore this specific
dependency's minor version or upgrade to it yourself)
- `@dependabot ignore <dependency name>` will close this group update PR
and stop Dependabot creating any more for the specific dependency
(unless you unignore this specific dependency or upgrade to it yourself)
- `@dependabot unignore <dependency name>` will remove all of the ignore
conditions of the specified dependency
- `@dependabot unignore <dependency name> <ignore condition>` will
remove the ignore condition of the specified dependency and ignore
conditions


</details>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

octo-sts[bot] committed:
[AutoDocs] Update Platform Docs (#2120)

Platform docs  autocommit

Signed-off-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: cpanato <4115580+cpanato@users.noreply.github.com>

SharpRake committed:
updating screenshots in registry authentication doc (#2121)

[X] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->
Fixes outdated screenshots in the registry authentication doc

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it.
-->
resolves https://github.com/chainguard-dev/internal/issues/4740

### Why are we making this change?
<!-- What larger problem does this PR address? -->

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list
criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please
add them. -->

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps.
-->
[preview
link](https://deploy-preview-2121--ornate-narwhal-088216.netlify.app/chainguard/chainguard-registry/authenticating/#managing-pull-tokens-in-the-chainguard-console)

Signed-off-by: Mark Drake <mark@chainguard.dev>

erikaheidi committed:
Fixing timestamps (#2125)

[X] Check if this is a typo or other quick fix and ignore the rest :)

There are some invalid timestamps that need to be fixed otherwise the
hugo build fails:

```
❯ hugo serve
Watching for changes in /home/erika/Projects/edu/{archetypes,assets,babel.config.js,content,data,layouts,node_modules,package.json,static}
Watching for config changes in /home/erika/Projects/edu/config/_default, /home/erika/Projects/edu/config/_default/menus
Start building sites … 
hugo v0.123.7+extended linux/amd64 BuildDate=2024-11-06T01:07:23Z VendorInfo=ubuntu:0.123.7-1ubuntu0.2

ERROR render of "page" failed: "/home/erika/Projects/edu/layouts/article/single.html:94:71": execute of template failed: template: article/single.html:94:71: executing "main" at <time.Format>: error calling Format: unable to parse date: 2025-03-0620T08:49:15+00:00
Built in 5339 ms
Error: error building site: render: failed to render pages: render of "page" failed: "/home/erika/Projects/edu/layouts/article/single.html:94:71": execute of template failed: template: article/single.html:94:71: executing "main" at <time.Format>: error calling Format: unable to parse date: 2025-03-0620T08:49:15+00:00

```

Our maintenance tool flagged these:

```
Building content catalog...
Catalog built with 314 items.

Found 7 articles with invalid timestamps.

Found 62 articles not updated in the last 7 months.

Title,Last Updated
/chainguard/chainctl-usage/chainctl-events,N/A
/chainguard/chainctl-usage/chainctl-iam,N/A
/chainguard/chainctl-usage/chainctl-images,N/A
/chainguard/chainctl-usage/chainctl-packages,N/A
/chainguard/chainctl-usage/chainctl-update,N/A
/chainguard/chainctl-usage/chainctl-version,N/A
/chainguard/chainctl-usage/getting-started-with-chainctl,N/A

```

This PR fixes the timestamps in these articles.

erikaheidi committed:
Adding dfc EA user guide to Academy (#2126)

This PR adds the [Dockerfile Conversion
Tool](https://github.com/chainguard-dev/dfc) EA user guide to Academy,
under the "Migration" menu, with the path
`/chainguard/migration/dockerfile-conversion/` .

In the moment, we only have a EA user guide, so this is a single
markdown file. We can expand this later on as a subfolder with multiple
docs.

Preview:
https://deploy-preview-2126--ornate-narwhal-088216.netlify.app/chainguard/migration/dockerfile-conversion/

---------

Signed-off-by: Erika Heidi <erikaheidi@users.noreply.github.com>
Co-authored-by: Mark Drake <33191761+SharpRake@users.noreply.github.com>

octo-sts[bot] committed:
[AutoDocs] Update Platform Docs (#2128)

Platform docs  autocommit

Signed-off-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: cpanato <4115580+cpanato@users.noreply.github.com>

jakeva committed:
Fix Several Typos (#2132)

[x] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it.
-->

### Why are we making this change?
<!-- What larger problem does this PR address? -->

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list
criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please
add them. -->

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps.
-->

Signed-off-by: Jake Valenzuela <jake.valenzuela@chainguard.dev>

octo-sts[bot] committed:
[AutoDocs] Update Platform Docs (#2131)

Platform docs  autocommit

Signed-off-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: cpanato <4115580+cpanato@users.noreply.github.com>

jakeva committed:
Capitalize 'AS' in Dockerfile examples (#2133)

[x] Check if this is a typo or other quick fix and ignore the rest :)

Align with [Dockerfile
conventions](https://docs.docker.com/reference/dockerfile/#format), and
closes #2127

ltagliaferri committed:
first updates (#2135)

[ ] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it.
-->

### Why are we making this change?
<!-- What larger problem does this PR address? -->

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list
criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please
add them. -->

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps.
-->

Signed-off-by: ltagliaferri <lisa.tagliaferri@gmail.com>
Signed-off-by: ltagliaferri <lisa@chainguard.dev>

eslerm committed:
pin github actions to full-length commit SHA (#2134)

Pin github actions to full-length commit SHAs.

Ref: https://github.com/chainguard-dev/prodsec/issues/60

ltagliaferri committed:
homepage update (#2137)

[ ] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it.
-->

### Why are we making this change?
<!-- What larger problem does this PR address? -->

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list
criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please
add them. -->

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps.
-->

---------

Signed-off-by: ltagliaferri <lisa.tagliaferri@gmail.com>

mosabua committed:
Add Chainguard Libraries for Java documentation (#2129)

## Type of change

Addition of new documentation

### What should this PR do?

Add new section "Libraries" for the documentation of Chainguard
Libraries for Java and future additions.

Implements https://github.com/chainguard-dev/internal/issues/4763

### Why are we making this change?

Need to provide info for users

### What are the acceptance criteria? 

* Review approval
* Wait for 25th of March, 5am PST at least before merge

### How should this PR be tested?

Read and review. All the chainctl and global config can be tested with a
local Nexus or Artifactory instance and a Maven build. I can also demo.

wlynch committed:
chainctl: fix cosign verify command (#2138)

I think the versioning output may have changed over time, so this was
returning `v0.1.2` rather than `0.1.2`, throwing off the URLs resulting
in the error:

Error: error during PEM decoding
main.go:74: error during command execution: error during PEM decoding

This fixes the version so the cert can be fetched.

[x] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it.
-->

### Why are we making this change?
<!-- What larger problem does this PR address? -->

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list
criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please
add them. -->

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps.
-->

SharpRake committed:
adding alias to fix 404 from blog link (#2139)

[ x ] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it.
-->

### Why are we making this change?
<!-- What larger problem does this PR address? -->

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list
criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please
add them. -->

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps.
-->

Signed-off-by: Mark Drake <mark@chainguard.dev>

dependabot[bot] committed:
Bump actions/setup-go from 5.3.0 to 5.4.0 in the actions group (#2136)

Bumps the actions group with 1 update:
[actions/setup-go](https://github.com/actions/setup-go).

Updates `actions/setup-go` from 5.3.0 to 5.4.0
<details>
<summary>Release notes</summary>
<p><em>Sourced from <a
href="https://github.com/actions/setup-go/releases">actions/setup-go's
releases</a>.</em></p>
<blockquote>
<h2>v5.4.0</h2>
<h2>What's Changed</h2>
<h3>Dependency updates :</h3>
<ul>
<li>Upgrade semver from 7.6.0 to 7.6.3 by <a
href="https://github.com/dependabot"><code>@​dependabot</code></a> in <a
href="https://redirect.github.com/actions/setup-go/pull/535">actions/setup-go#535</a></li>
<li>Upgrade eslint-config-prettier from 8.10.0 to 10.0.1 by <a
href="https://github.com/dependabot"><code>@​dependabot</code></a> in <a
href="https://redirect.github.com/actions/setup-go/pull/536">actions/setup-go#536</a></li>
<li>Upgrade <code>@​action/cache</code> from 4.0.0 to 4.0.2 by <a
href="https://github.com/aparnajyothi-y"><code>@​aparnajyothi-y</code></a>
in <a
href="https://redirect.github.com/actions/setup-go/pull/568">actions/setup-go#568</a></li>
<li>Upgrade undici from 5.28.4 to 5.28.5 by <a
href="https://github.com/dependabot"><code>@​dependabot</code></a> in <a
href="https://redirect.github.com/actions/setup-go/pull/541">actions/setup-go#541</a></li>
</ul>
<h2>New Contributors</h2>
<ul>
<li><a
href="https://github.com/aparnajyothi-y"><code>@​aparnajyothi-y</code></a>
made their first contribution in <a
href="https://redirect.github.com/actions/setup-go/pull/568">actions/setup-go#568</a></li>
</ul>
<p><strong>Full Changelog</strong>: <a
href="https://github.com/actions/setup-go/compare/v5...v5.4.0">https://github.com/actions/setup-go/compare/v5...v5.4.0</a></p>
</blockquote>
</details>
<details>
<summary>Commits</summary>
<ul>
<li><a
href="https://github.com/actions/setup-go/commit/0aaccfd150d50ccaeb58ebd88d36e91967a5f35b"><code>0aaccfd</code></a>
Bump undici from 5.28.4 to 5.28.5 (<a
href="https://redirect.github.com/actions/setup-go/issues/541">#541</a>)</li>
<li><a
href="https://github.com/actions/setup-go/commit/c4c114188661c0fa735e5b938764519fc6e8efa9"><code>c4c1141</code></a>
upgrade actions/cache to 4.0.2 (<a
href="https://redirect.github.com/actions/setup-go/issues/568">#568</a>)</li>
<li><a
href="https://github.com/actions/setup-go/commit/5a083d0e9a84784eb32078397cf5459adecb4c40"><code>5a083d0</code></a>
Bump eslint-config-prettier from 8.10.0 to 10.0.1 (<a
href="https://redirect.github.com/actions/setup-go/issues/536">#536</a>)</li>
<li><a
href="https://github.com/actions/setup-go/commit/1d82324e5352acd1afdae34b93def4fabce6599f"><code>1d82324</code></a>
Bump semver from 7.6.0 to 7.6.3 (<a
href="https://redirect.github.com/actions/setup-go/issues/535">#535</a>)</li>
<li>See full diff in <a
href="https://github.com/actions/setup-go/compare/f111f3307d8850f501ac008e886eec1fd1932a34...0aaccfd150d50ccaeb58ebd88d36e91967a5f35b">compare
view</a></li>
</ul>
</details>
<br />


[![Dependabot compatibility
score](https://dependabot-badges.githubapp.com/badges/compatibility_score?dependency-name=actions/setup-go&package-manager=github_actions&previous-version=5.3.0&new-version=5.4.0)](https://docs.github.com/en/github/managing-security-vulnerabilities/about-dependabot-security-updates#about-compatibility-scores)

Dependabot will resolve any conflicts with this PR as long as you don't
alter it yourself. You can also trigger a rebase manually by commenting
`@dependabot rebase`.

[//]: # (dependabot-automerge-start)
[//]: # (dependabot-automerge-end)

---

<details>
<summary>Dependabot commands and options</summary>
<br />

You can trigger Dependabot actions by commenting on this PR:
- `@dependabot rebase` will rebase this PR
- `@dependabot recreate` will recreate this PR, overwriting any edits
that have been made to it
- `@dependabot merge` will merge this PR after your CI passes on it
- `@dependabot squash and merge` will squash and merge this PR after
your CI passes on it
- `@dependabot cancel merge` will cancel a previously requested merge
and block automerging
- `@dependabot reopen` will reopen this PR if it is closed
- `@dependabot close` will close this PR and stop Dependabot recreating
it. You can achieve the same result by closing it manually
- `@dependabot show <dependency name> ignore conditions` will show all
of the ignore conditions of the specified dependency
- `@dependabot ignore <dependency name> major version` will close this
group update PR and stop Dependabot creating any more for the specific
dependency's major version (unless you unignore this specific
dependency's major version or upgrade to it yourself)
- `@dependabot ignore <dependency name> minor version` will close this
group update PR and stop Dependabot creating any more for the specific
dependency's minor version (unless you unignore this specific
dependency's minor version or upgrade to it yourself)
- `@dependabot ignore <dependency name>` will close this group update PR
and stop Dependabot creating any more for the specific dependency
(unless you unignore this specific dependency or upgrade to it yourself)
- `@dependabot unignore <dependency name>` will remove all of the ignore
conditions of the specified dependency
- `@dependabot unignore <dependency name> <ignore condition>` will
remove the ignore condition of the specified dependency and ignore
conditions


</details>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

erikaheidi committed:
Adding JRE + Minecraft tutorial (#2140)

This PR adds the JRE + Minecraft tutorial to Academy, under thee
"Getting Started" image guides.

The content has been previously reviewed by @SharpRake on GDocs, so I am
assigning him this review for a quick check since he has all the context
needed.

Preview Link:
http://localhost:1313/chainguard/chainguard-images/getting-started/jre-minecraft/

---------

Signed-off-by: Erika Heidi <erikaheidi@users.noreply.github.com>
Co-authored-by: Mark Drake <33191761+SharpRake@users.noreply.github.com>

smythp commented on content/chainguard/administration/assumable-ids/identity-examples/aws-ec2-identity.md:
````diff
@@ -0,0 +1,206 @@
+Finally, use this token — along with the `$CHAINGUARD_IDENTITY` variable — to authenticate to Chainguard from your EC2 instance:
+
+```EC2
+chainctl auth login  --identity-token $TOK --identity $CHAINGUARD_IDENTITY
````
I got this at this stage:

```
Error: [101] unable to exchange tokens: rpc error: code = Unknown desc = aws arn mismatch

```

SharpRake replied:
Hmm, I tried it again on a fresh EC2 instance and it worked. I will publish this doc with your other suggestions and we'll see what kind of feedback we get.

smythp commented on content/chainguard/administration/assumable-ids/identity-examples/aws-ec2-identity.md:
```diff
@@ -0,0 +1,206 @@
+
+## Learn more
+
+By following this guide, you will have created a Chainguard identity that you can use to authenticate to Chaingaurd from an Amazon EC2 instance. For more information about how assumable identities work in Chainguard, check out our [conceptual overview of assumable identities](/chainguard/administration/iam-organizations/assumable-ids/). Additionally, we encourage you to read through the rest of o ...
```
```suggestion
By following this guide, you will have created a Chainguard identity that you can use to authenticate to Chainguard from an Amazon EC2 instance. For more information about how assumable identities work in Chainguard, check out our [conceptual overview of assumable identities](/chainguard/administration/iam-organizations/assumable-ids/). Additionally, we encourage you to read through the rest of our documentation on [Administering Chainguard resources](/chainguard/administration/).
```

smythp reviewed (approved):
Approving, though I had a token mismatch in the final step. Let me know if you want to sync and run through it next week, up to you.

amouat committed:
Fix typo (#2144)

[ x] Check if this is a typo or other quick fix and ignore the rest :)

Signed-off-by: Adrian Mouat <adrian@chainguard.dev>

mosabua committed:
Improve Chainguard Libraries docs

mosabua committed:
Refer to Chainguard Factory as internal system

[x] Check if this is a typo or other quick fix and ignore the rest :)

Follow up to #2141 as requested by @ltagliaferri

SharpRake committed:
PS's suggestions

Signed-off-by: Mark Drake <mark@chainguard.dev>

SharpRake commented:
This PR is completely borked, closing in favor of this one: https://github.com/chainguard-dev/edu/pull/2150
